community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v3_3/TransactionBoundPlanContext.scala

```diff
@@ -190,6 +190,7 @@ class TransactionBoundPlanContext(tc: TransactionalContextWrapper, logger: Inter
     case Neo4jTypes.NTGeometry => symbols.CTGeometry
     case Neo4jTypes.NTMap => symbols.CTMap
     case Neo4jTypes.NTAny => symbols.CTAny
+    case Neo4jTypes.NTByteArray => symbols.CTList(symbols.CTInteger)
```

Comment:
I think you need to handle this in old versions as well,  so I guess v3_2 and v3_1 needs to somehow cope with this new type

Reply:
Interestingly we have no compatibiity tests for calling procedures using older versions of Cypher. I suspect this is not a well supported area. I think we should handle that as a separate PR, testing more than just byte[].